### DISCUSSION TITLE
Run x86_64 AppImages on Raspberry Pi 5

### DISCUSSION THREAD

**Discussion by probonopd** [ptitSeb/box64#1078] [Show and tell]
Many [AppImages](https://appimage.github.io/) are built for x86_64 (Intel/AMD) systems, but thanks to box64 these can run nicely on 64-bit ARM machines such as the Raspberry Pi 5. Best of all, if box64 is [installed](https://raw.githubusercontent.com/Botspot/pi-apps/master/apps/Box64/install-64), this works seamlessly - just double click AppImages as usual. Mind blown!

![image](https://github.com/ptitSeb/box64/assets/2480569/e99ec91a-f886-483b-a8d2-9ff7e290763b)

Mind blown.

Details: https://github.com/orgs/AppImage/discussions/1294

**Comment by Inqurity** [comment]
![obraz](https://github.com/user-attachments/assets/b66c89e5-6864-41b5-9b75-319895dacbc0)
Didn't work quite as smoothly for me
